doc/install.rst

```diff
@@ -5,36 +5,33 @@
 Installing and Getting Started
 ==============================
 
-Installing PySurfer is quite simple with pip_::
+PySurfer can be installed with pip_. Note that the package name on PyPi is different from the library name that you import::
 
     pip install pysurfer
 
 If you already have PySurfer installed, you can also use pip to update it::
 
     pip install -U pysurfer
 
-If you would like to save movies of time course data, also include the
-optional dependency `imageio` with::
+If you would like to save movies of time course data, it is necessary to include the optional dependency ``imageio`` with::
 
     pip install -U pysurfer[save_movie]
 
 If you'd like to install the development version, you have two options. You can
-use pip::
+install straight from github::
 
-    pip install git+git://github.com/nipy/pysurfer.git#egg=pysurfer
+    pip install git+https://github.com/nipy/pysurfer.git
```

Comment:
use:

`pip install -U https://api.github.com/repos/nipy/PySurfer/zipball/master`

so no need for git which typically a pb on windows...